src/svelte-asset.js

```diff
@@ -34,40 +40,46 @@ class SvelteAsset extends Asset {
     this.type = 'js';
   }
 
-  async generate() {
-    let compilerOptions = {
+  async getConfig() {
+    let config = (await super.getConfig(['.svelterc', 'svelte.config.js', 'package.json'])) || {};
+    config = config.svelte || config;
+
+    let defaultOptions = {
       generate: 'dom',
-      format: 'cjs',
       store: true,
-      css: true
+      css: true,
+      format: 'cjs'
```

Comment:
Defaulting this to es || esm didn't work :(

Reply:
Strange. The Svelte 3 compiler only recognises `esm`, `es` is v2. Maybe it's the same issue as discussed with dependency versions

Reply:
Not entirely sure why it happens but it has something to do with the HMR code, probably the wrapper that breaks imports

Reply:
I think I figured it out, the regex in makeHot was only working for commonjs